app/javascript/admin/displayErrorAlert.js

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
This function can be imported and used within any StimulusJS controller in Admin to generate error alerts.

Reply:
nitpick (non-blocking): Consolidate this function and `displaySnackbar` in one file and name it something along the lines of `messageUtilities`.